test_util/test_upgrade_vpc.py

```diff
@@ -249,6 +268,15 @@ def marathon_app_tasks_survive_upgrade():
                         "test_upgrade_vpc.marathon_app_tasks_survive_upgrade",
                         details="expected: {}\nactual:   {}".format(self.tasks_start, tasks_end))
 
+            def test_task_state_remains_consistent():
```

Comment:
minor: renamed test to `test_mesos_task_state_remains_consistent` since most things have something to do with tasks, having the extra mesos piece will help keep things grouped.